lib/editor/components/map/LocationsLayer.js

```diff
@@ -95,7 +95,7 @@ const LocationsLayer = (props: Props) => {
     const locationShapes = clone(activeEntity.location_shapes)
     const {key} = e
 
-    const newLocationShapes = locationShapes.filter((shapePt) => !(key === shapePt.geometry_id))
+    const newLocationShapes = locationShapes.filter((shapePt) => !(Number(key) === shapePt.geometry_id))
```

Comment:
Serious nitpick: do you think `parseInt` makes more sense? I didn't even know about `Number` so maybe this is just me not staying up to date with js best practices...

Reply:
@miles-grant-ibigroup This is a great question, I didn't consider `parseInt`... I _think_ since the goal is type conversion, `Number` might be more appropriate? But in this case they should be functionally the same, so I'm happy to change it! Going off of this: https://stackoverflow.com/questions/4090518/what-is-the-difference-between-parseint-and-number

Reply:
That was helpful thanks! Let's stick with number then I guess!

Reply:
Thanks so much!!